libr/cons/pal.c

```diff
@@ -457,6 +457,9 @@ static void r_cons_pal_show_rgb() {
 R_API void r_cons_pal_show() {
 	int i;
 	for (i = 0; colors[i].name; i++) {
+		if (r_cons_singleton ()->ansicon && !strcmp (colors[i].name, "gray")) {
```

Comment:
that will hide not just the color but also the entry in the palette. it is correct?

Reply:
no, the entry is still in the palette a.k.a. `colors` table

Reply:
it can't be removed since at least the `basic` eco theme uses it

Reply:
the entry for gray in the `colors` table should be changed if ansicon, but that's a different pr